website/content/docs/commands/validate.mdx

```diff
@@ -31,6 +31,12 @@ Errors validating build 'vmware'. 1 error(s) occurred:
 - `-syntax-only` - Only the syntax of the template is checked. The
   configuration is not validated.
 
+- `-evaluate-datasources` - Evaluate the datasources when validating the template.
+  This is only valid on HCL2 templates, since JSON templates do not feature
+  datasources, this option will be ignored. Note that because datasources may rely
+  on external services for fetching data, this will incur some costs at validation
+  when the services being contacted are billing for the operation.
+
```

Comment:
Slight reword with a call out for the billing warning 
```suggestion
- `-evaluate-datasources` - Evaluate all data sources when validating a template.
  This is only valid on HCL2 templates, since JSON templates do not feature
  data sources. For JSON templates this flag will be ignored. 
  
  ~> **Warning:** Data sources may rely on external services for fetching data, which can incur some costs at validation if the services being contacted are billing per operation.
```